src/tip.ts

```diff
@@ -95,7 +95,7 @@ export async function tipUser(
         unsub();
       }
     });
-
+  await api.disconnect();
```

Comment:
does this solve the reconnection or runtime upgrade issue?

Reply:
No, But I am hoping this will solve the memory leak issue that was reported.